tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -368,6 +419,17 @@ class CompletionSuite extends BaseCompletionSuite {
       "2" ->
         """|ProcessBuilder java.lang
            |ProcessBuilder - scala.sys.process
+           |CertPathBuilder - java.security.cert
+           |CertPathBuilderSpi - java.security.cert
+           |ProcessBuilderImpl - scala.sys.process
+           |CertPathBuilderResult - java.security.cert
+           |PKIXBuilderParameters - java.security.cert
+           |PooledConnectionBuilder - javax.sql
+           |CertPathBuilderException - java.security.cert
+           |PKIXCertPathBuilderResult - java.security.cert
+           |""".stripMargin,
+      ">=3.4.1-RC1-bin-20240201-hash-NIGHTLY" ->
+        """|ProcessBuilder - scala.sys.process
```

Comment:
Why did ProcessBuilder disappear?

Reply:
It is in scope, and compiler doesn't support fuzzy search. I have a fix for that that will ship next week.